### DISCUSSION TITLE
Browsershot + ApexCharts: I get no chart in PDF

### DISCUSSION THREAD

**Discussion by cjaoude** [spatie/browsershot#467] [Q&A]
It's been a bit frustrating....hoping someone has come across this. Using a clean-sheet Laravel app with very basic view+chart, ApexCharts won't render anything in PDF. The chart renders just fine in a normal view, but piping the view through `Browsershot::html($view)->savePdf('example.pdf')` nothing happens. I've tried `->waitUntilNetworkIdle()` and `waitForFunction` but nothin'. 

Insights appreciated. 👍

**Comment by RikoDEV** [comment]
I have exactly the same problem. If I use Browsershot::html() instead of Browsershot::url() no javascript is executed.
